### DISCUSSION TITLE
Hello_hera.py for version 5.1.3 ?

### DISCUSSION THREAD

**Discussion by EricBoix** [argoproj-labs/hera#581] [Q&A] [answered]
The Hera version [5.1.3 examples](https://github.com/argoproj-labs/hera/tree/5.1.3/examples/workflows) do not seem to provide the equivalent of [`hello_hera.py` of version 4.4.2](https://github.com/argoproj-labs/hera/blob/4.4.2/examples/hello_hera.py).

I tried the following direct adaptation
```python
from hera.workflows import Workflow, Task

def hello():
    print("Hello, Hera!")

with Workflow(generate_name="hello-world-") as w:
    Task(name="pythonprint", hello)
w.create()
```

that alas fails with `TypeError: __init__() takes exactly 1 positional argument (2 given)`.

Additionally it seems that 
- the [source code of class Task()](https://github.com/argoproj-labs/hera/blob/5.1.3/src/hera/workflows/task.py) doesn't include anymore the complete constructor documentation that was [available for version 4.4.2](https://github.com/argoproj-labs/hera/blob/4.4.2/src/hera/task.py#L80)
- and they are very few "real" examples using a `Task()` constructor with constructor arguments. 

Any hint ?

**Comment by sambhav** [comment, marked as answer]
@EricBoix You can find a simple example at https://hera.readthedocs.io/en/stable/examples/workflows/global_config/

It can be shortened to - 

```py
from hera.workflows import Workflow, script

@script()
def say():
    print("hello")


with Workflow(generate_name="hello-", entrypoint="say") as w:
    say()
w.create()
```

**Comment by EricBoix** [comment]
Thanks @samj1912. Indeed, the syntax did change a lot from 4.4.2. 
And thanks for pointing out the existence of the nice documentation 
(I had missed to click on the docs badge when on the github landing page: my bad).
Closing.
